api-playground/mdx/authentication.mdx

```diff
@@ -28,36 +30,58 @@ authMethod: "bearer"
 
 </CodeGroup>
 
-## Supported Authentication Methods
-
-- bearer
-- basic
-- none
-
-The "none" authentication method is useful to disable authentication on a specific endpoint after setting a default in mint.json.
-
-## Authentication Name
+### Basic Authentication
 
-By default, basic authentication asks for username and password. However, basic authentication can use a different name for username and password. For example, you could have called it `user-id:user-key`.
-
-You can set the name property in mint.json to override the default functionality. Use colons to separate each property you want to request.
+<CodeGroup>
 
-```json Custom username and password
+```json mint.json
 "api": {
     "auth": {
-        "method": "basic",
-        "name": "user-id:user-key"
+        "method": "basic"
     }
 }
 ```
 
-You can also request a single API key by excluding the colon.
+```md page metadata
```

Comment:
could we capitalize all instances of `Page Metadata`?

Reply:
yes! globally or just this diff?

![image](https://github.com/user-attachments/assets/a14e20b2-a907-4c68-8f1a-be8412c19af2)

Reply:
just the ones that are ```md page metadata on this page

Reply:
updated!

![image](https://github.com/user-attachments/assets/ecb2f81a-eb4c-412c-9ffb-def8b5109271)